bin/main.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+require_relative '../lib/board.rb'
+require_relative '../lib/player.rb'
+require 'colorize'
+
+weapon = ''
+input = false
+game = Board.new
+
+puts 'Welcome to Tic-Tac-Toe game'
+puts 'Press Enter to continue'
+gets
+
+puts 'Enter name for player 1'
+name = gets.chomp
+
+until weapon == 'X' || weapon == 'O'
```

Comment:
As linter suggests, you should use an array, write it like `['X', 'O'].include?(weapon)`